tools/flang2/flang2exe/iliutil.cpp

```diff
@@ -2614,6 +2614,9 @@ addarth(ILI *ilip)
   case IL_IDIM:
```

Comment:
It would be great if you could combine changes in this file with the changes proposed in (rejected) PR #503 (the file was named iliutil.c back then, but the change still applies more or less cleanly). We're pulled in this change downstream long time ago, and it improves utilization of libpgmath functions on AArch64.

Reply:
Thanks for linking that PR. I think it's a valuable patch and, if it is currently abandoned, I can try to find time to resurrect the `iliutil.cpp` parts, in a future PR. But I don't think it should be combined with this patch, which should be strictly about quad-precision FP support.